model/eventGroupModel.js

```diff
@@ -3,19 +3,19 @@ const Mongoose = require('mongoose');
 const eventGroupSchema = Mongoose.Schema({
     eventID: {
         type: Mongoose.Types.ObjectId,
-        required: true
+        required: [true, 'Event ID must be specified']
     },
     companyID: {
         type: Mongoose.Types.ObjectId,
-        required: true
+        required: [true, 'Company ID must be specified']
     },
     name: {
         type: String,
-        required: true
+        required: [true, 'Group name must be specified']
```

Comment:
Add `min` and `maxlength` checks. I do not know if `minlength` is needed since you have `required: true`, but I don't know if you can send an empty string and it will treat it as valid since it tehnically is there? Maybe check that edge-case as well.